corehq/apps/app_manager/static/app_manager/js/report-module.js

```diff
@@ -0,0 +1,95 @@
+
+var ReportModule = (function () {
+
+    function ReportConfig(report_id, display, availableReportIds, language) {
+        var self = this;
+        this.lang = language;
+        this.fullDisplay = display || {};
+        this.availableReportIds = availableReportIds;
+        this.display = ko.observable(this.fullDisplay[this.lang]);
```

Comment:
Nice! My first though on looking at the screenshot was "Hope he remembered to split display out by language". Would be a big pain to have to add later.